homeassistant/components/imgw_pib/config_flow.py

```diff
@@ -45,7 +45,9 @@ async def async_step_user(
 
             try:
                 imgwpib = await ImgwPib.create(
-                    client_session, hydrological_station_id=station_id
+                    client_session,
+                    hydrological_station_id=station_id,
+                    hydrological_details=False,
```

Comment:
Adding missing `hydrological_details` parameter, relates to https://github.com/home-assistant/core/pull/134668